openshift_metrics/openshift_prometheus_metrics.py

```diff
@@ -0,0 +1,65 @@
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+#
+
+import argparse
+import datetime
+import os
+
+import openshift as oc
+
+import utils
+
+
+CPU_METRIC = 'kube_pod_init_container_resource_requests_cpu_cores'
+ALLOCATED_CPU_METRIC = 'kube_pod_init_container_resource_limits_cpu_cores'
+ALLOCATED_MEMORY_METRIC = 'kube_pod_init_container_resource_limits_memory_bytes'
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--openshift-url", help="OpenShift Prometheus URL")
+    parser.add_argument("--report-date", help="report date (ex: 2022-03-14)")
+
+    args = parser.parse_args()
+    if args.openshift_url:
+        openshift_url = args.openshift_url
+    else:
+        if not os.getenv('OPENSHIFT_PROMETHEUS_URL'):
+            raise Exception('Must specify OPENSHIFT_PROMETHEUS_URL in your environment')
+        openshift_url = os.getenv('OPENSHIFT_PROMETHEUS_URL')
+    if args.report_date:
+        report_date = args.report_date
+    else:
+        report_date = (datetime.datetime.today() - datetime.timedelta(days=1)).strftime('%Y-%m-%d')
```

Comment:
This is something else you could potentially handle by adding a default value to the `add_argument` call.

Reply:
Fixed as well!